go/vt/sqlparser/parse_test.go

```diff
@@ -559,9 +559,11 @@ var (
 	}, {
 		input: "select /* not like escape */ 1 from t where a not like b escape '$'",
 	}, {
-		input: "select /* regexp */ 1 from t where a regexp b",
+		input:  "select /* regexp */ 1 from t where a regexp b",
+		output: "select /* regexp */ 1 from t where a regexp b",
```

Comment:
Input and output are the same here, so I think we can drop the output here to reduce the diff?